migrations/20220105170001-migrate-past-platformTip-orders.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(
+      `
+        UPDATE
+          "Orders"
+        SET
+          "platformTipAmount" = ("data"->'platformFee')::int,
+          "platformTipEligible" = true
```

Comment:
Sure, platformTipEligible is TRUE if a platformTip is set. Would be interesting to populate it when there is no platformTip.

Why not try to do it like suggested in https://github.com/opencollective/opencollective-api/pull/7019#discussion_r780403540 ?